store/tikv/rawkv.go

```diff
@@ -313,6 +313,10 @@ func (c *RawKVClient) Scan(startKey, endKey []byte, limit int) (keys [][]byte, v
 		if len(startKey) == 0 {
 			break
 		}
+
+		if len(endKey) > 0 && bytes.Compare(startKey, endKey) > 0 {
```

Comment:
I think `bytes.Compare(startKey, endKey) > 0` should be `bytes.Compare(startKey, endKey) >= 0`. And I would like to move the check before sending request:
```
for len(keys) < limit && (len(endKey)==0 || bytes.Compare(startKey, endKey) >= 0) {
  ...
}
```
In this way if startKey >= endKey, we don't need to send any requests.

Reply:
ok